src/view.branch.js

```diff
@@ -0,0 +1,61 @@
+class BranchSwitch {
+  constructor($dom, store) {
+    this.$dom = $dom;
+    this.store = store;
+    this.branches = [];
+    $dom.find('.octotree-views').on('click', '.octotree-header-branch', this._showPopUp.bind(this));
+  }
+
+  setBranches(branches) {
+    if (!this.branches.length) {
+      this.branches = branches;
+    }
+  }
+
+  _showPopUp() {
+    $('html')
+      .append(
+        `
+           <div class='modal-root'>
```

Comment:
could we mimic the dropdown of Github branches switching for the sake of consistency ?